<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1758</width>
    <height>1015</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>1091</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Truyền thông:</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>211</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Cổng hiện có:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_port">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>COM 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM 7</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_baud">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>140</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_bitstop">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_databit">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>140</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_direction">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>70</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Input</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Output</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Input|Output</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_partiy">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>70</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NoParity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EvenParity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OddParity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SpaceParity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MarkParity</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_flowcontrol">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>140</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NoFlowControl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HardwareControl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SoftwareControl</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_uart">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>120</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>CONNECT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_port">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PORT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_databit">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DataBit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_baud">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Baud</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_bitstop">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>BitStop</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_direction">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Direction</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_flowcontrol">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FlowControl</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_parity">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Parity</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mesUart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ket noi</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto check connect</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDTR">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>70</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>DTR</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxProcessingDelay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>211</width>
       <height>26</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the interval of reading Serial/UDP. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_gui">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nhập dữ liệu gửi ( ấn 1 lần: gửi | ấn 2 lần: nhận dữ liệu): (lỗi time)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gui">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>331</width>
      <height>851</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditsend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>tin hieu da ket noi vao file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Messenge:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>1391</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>simulink</string>
     </attribute>
     <widget class="QWidget" name="widgetchart" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1311</width>
        <height>691</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>shchematic</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1758</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest_System">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_q">
    <property name="title">
     <string>đăq</string>
    </property>
   </widget>
   <addaction name="menuTest_System"/>
   <addaction name="menu_q"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
